CRM/Core/Resources/Common.php

```diff
@@ -129,7 +129,7 @@ public static function createFullBundle($name) {
       if (is_array($item)) {
         $bundle->addSetting($item);
       }
-      elseif (strpos($item, '.css')) {
+      elseif (substr($item, -4) == '.css') {
```

Comment:
Maybe `parse_url` is the most correctest way?
```suggestion
      elseif (substr(parse_url($item, PHP_URL_PATH), -4) === '.css') {
```